cypress/e2e/spec.cy.ts

```diff
@@ -192,7 +192,10 @@ describe("When exporting and/or importing a board", () => {
     cy.get('[data-testid="DownloadIcon"]').click()
 
     const date = new Date()
-    const fileName = `${defaultBoard.title}-${date.toLocaleString("fr-FR", { day: "2-digit", month: "2-digit", year: "numeric" }).replace(/[^a-zA-Z0-9]/g, "_")}.csv`
+    const dateString = date
+      .toLocaleString("fr-FR", { day: "2-digit", month: "2-digit", year: "numeric" })
```

Comment:
If we get more of these, we should just install day.js to make this cleaner, but because this is currently the only one, there is no point in adding another library